tensorboard/util/BUILD

```diff
@@ -79,6 +79,17 @@ py_library(
     srcs_version = "PY2AND3",
 )
 
+py_library(
+    name = "tensor_manip",
+    srcs = ["tensor_manip.py"],
```

Comment:
Actually - how do you feel about naming this `tensor_util` instead?  If it can be done via a big find-replace hopefully it's not too bad.

That matches the original name of the file in TF (I don't know why the stub PR named it `tensor_manip` instead) and I think `util` is a bit clearer (it's a more standard abbreviation than "manip" and it communicates more generally that these are small utilities rather than general-purpose tensor manipulation).

Reply:
Done!